specifications/api/swagger.yaml

```diff
@@ -2138,12 +2693,37 @@ x-responsesGetTransaction: &responsesGetTransaction
         schema: *ApiTransaction
 
 x-responsesPostTransaction: &responsesPostTransaction
-  <<: *responsesErr400
-  <<: *responsesErr403
-  <<: *responsesErr404
-  <<: *responsesErr405
+  400:
+    description: Bad Request
+    content:
+      application/json:
+        schema:
+          oneOf:
+            - <<: *errBadRequest
+            - <<: *errTransactionIsTooBig
```

Comment:
I think this should be only in `403`?
![Screenshot from 2020-10-30 11-10-06](https://user-images.githubusercontent.com/42900201/97692793-7f6ef600-1aa0-11eb-82eb-a5765196ce7b.png)

Reply:
```
        ErrSelectForPaymentTxTooLarge maxSize maxN  ->
            apiError err400 TransactionIsTooBig $ mconcat
                [ "I am afraid that the transaction you're trying to submit is "
                , "too large! The network allows transactions only as large as "
                , pretty maxSize, "s! As it stands, the current transaction only "
                , "allows me to select up to ", showT maxN, " inputs. Note "
                , "that I am selecting inputs randomly, so retrying *may work* "
                , "provided I end up choosing bigger inputs sufficient to cover "
                , "the transaction cost. Alternatively, try sending to less "
                , "recipients or with smaller metadata."
                ]
```
it is 400